platform_versioned_docs/version-23.4.0/data/data-studios.mdx

```diff
@@ -167,3 +174,8 @@ If you have access to the AWS Console for your organization, check the jobs asso
 ### What happens if the memory allocation of the data studio is exceeded?
 
 The running container in the AWS Batch compute environment inherits the memory limits specified by the data studio configuration when adding or starting the session. The kernel then handles the memory as if running natively on Linux. Linux can overcommit memory, leading to possible out-of-memory errors in a container environment. The kernel has protections in place to prevent this, but it can happen, and in this case, the process is killed. This can manifest as a performance lag, killed subprocesses, or at worst, a killed data studio session.
+
+### All my datasets are read-only
+
+By default, AWS Batch compute environments created with Batch Forge restrict access to S3 to the working directory only unless additional Allowed S3 Buckets are specified.
```

Comment:
Is _Allowed S3 Buckets_ a UI element?

Reply:
It is and it is also referred to in other parts of the documentation see https://docs.seqera.io/platform/23.4.0/compute-envs/aws-batch :)

Reply:
@jimmypoms, looks like we've deprecated Fusion v1, though. Should we suggest people rely on this feature to have access to additional buckets? I'm not familiar with any of the background on this.